rollup/rollup.browser.mjs

```diff
@@ -1,22 +1,29 @@
 import nodePolyfills from "rollup-plugin-polyfill-node";
 import { wasm } from "@rollup/plugin-wasm";
 import Base from "./base.mjs";
-import modify from "rollup-plugin-modify";
 import nodeResolve from "@rollup/plugin-node-resolve";
-
+import stripCode from "rollup-plugin-strip-code"
+import copy from 'rollup-plugin-copy'
+import commonjs from "@rollup/plugin-commonjs";
 export default Base({ mode: "browser" }, [
-  modify({
-    find: '"didcomm-node"',
-    replace: '"didcomm-browser"',
-  }),
-  nodePolyfills(),
   nodeResolve({
-    resolveOnly: ["anoncreds-browser", "didcomm-browser"],
+    resolveOnly: ["didcomm-browser", "anoncreds-browser"],
+  }),
+  copy({
+    targets: [
+      { src: "./generated/anoncreds-wasm-browser/anoncreds_bg.wasm", dest: "build/browser" },
+      { src: "./generated/didcomm-wasm-browser/didcomm_js_bg.wasm", dest: "build/browser" },
+    ],
```

Comment:
doesn't this conflict with the `wasm` plugin below?

Reply:
What u mean, do u see it conflicting?

Reply:
This is needed, because i don't want the users to have access to generated folder. 
This fixes issues like the following: